ecosystem/sep-0010.md

```diff
@@ -94,21 +96,19 @@ On success the endpoint must return `200 OK` HTTP status code and a JSON object
   * source account set to server's signing account
   * invalid sequence number (set to 0) so the transaction cannot be run on the Stellar network
   * time bounds: `{min: now(), max: now() + 900 }` (we recommend expiration of 15 minutes to give user time to sign transaction)
-  * operations: `manage_data(source: client_account, key: '<anchor name> auth', value: random_nonce())`
-    * The value of key is not important, but can be the name of the anchor followed by `auth`. It can be at most 64 bytes.
-    * The value must be 64 bytes long. It contains a 48 byte cryptographic-quality random string encoded using base64 (for a total of 64 bytes after encoding).
-  * signature by the web service signing account
+  * operations: `manage_data(source: client_account, key: '<anchor name> auth', value: '<home domain>')`
```

Comment:
Should the length limit for `home_domain` be stated here? I think this limit may be triggered.

Reply:
It's possible but I would think unlikely. Here is a string that would reach the 64-byte limit:
```
>>> from base64 import b64encode
>>> len(b64encode(b"thisisatest.sandbox.anchor.anchordomain.com auth"))
64
```

I'll add a note about the limit though.